src/v/pandaproxy/api/api-doc/schema_registry_header.json

```diff
@@ -2,7 +2,7 @@
   "swagger": "2.0",
   "info": {
     "title": "Pandaproxy Schema Registry",
-    "version": "1.0.0"
+    "version": "1.0.1"
```

Comment:
Just curious: is this version type exposed anywhere?

Reply:
```
curl -s localhost:8081/v1 | jq '.info.version'`
"1.0.1"
```

Reply:
Also, I should set up a better home for [the swagger](https://app.swaggerhub.com/apis/BenPope/pandaproxy-schema-registry/1.0.1)